.gitignore

```diff
@@ -1,4 +1,4 @@
 phpunit.xml
 vendor/
 composer.lock
-
+.idea/
```

Comment:
this should not be here. It is specific to an IDE, not to the project. You should rely on system-specific ignore rules instead: https://help.github.com/articles/ignoring-files/